src/josegonzalez/Queuesadilla/Worker/SequentialWorker.php

```diff
@@ -2,11 +2,28 @@
 
 namespace josegonzalez\Queuesadilla\Worker;
 
+declare(ticks=1);
```

Comment:
Do we actually need this? iirc `ticks` are only necessary for versions under 5.3, which this project cannot support.